### PR TITLE
docs: Document how to create and build ABC-only tests

### PR DESCRIPTION
**Pull request by kmeisthax**
This lightly documents the rabcasm tools that we use for tests that need specific bytecode sequences and aren't (fully) written in AS3. Some of our tests already use this, but we haven't explained this properly - we already had one PR that wound up submitting just SWFs and no source for the tests because of this.

Sent from my iPad

### CODE REVIEW

**Comment by midgleyc** [comment]
You'll probably want to remove the bit I added as part of my PR:

> For opcode tests involving running particular opcodes, you can use [`RABCDAsm`](https://github.com/CyberShadow/RABCDAsm) to deassemble an existing SWF, change the generated asasm, then reassemble.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -145,6 +145,11 @@ To add a test here, create a .swf file that runs `trace()` statements. You can d
     * [`mxmlc`](https://helpx.adobe.com/air/kb/archived-air-sdk-version.html) (ActionScript 3 only)
         * if you create a file `test.as` with a `class Test`, you can compile it using `mxmlc Test.as`. `mxmlc` is located in the `bin` folder of the downloadable AIR SDK.
         * you may want to use docker instead -- something like `docker run -it --rm -v ${PWD}:/src jeko/airbuild mxmlc ./Test.as` works well
+    * [`rabcasm`](https://github.com/CyberShadow/RABCDAsm) (ActionScript 3 only)
+    	* This test allows writing AVM2 bytecode sequences directly, without intermediate AS3 code. This is primarily useful for testing opcodes that aren't generated by the above-mentioned AS3 compilers.
+    	* `rabcasm` cannot generate SWF files from scratch. Instead, you must first generate a SWF from the above mentioned methods, then extract and disassemble it's ABC with `abcexport` and `rabcdasm`.
```
Comment (midgleyc) [inline CONTRIBUTING.md:150]: its

**Comment by kmeisthax** [comment]
Aah thanks, I didn't notice it 😅

**Comment by midgleyc** [comment]
LGTM!

**Comment by relrelb** [comment]
Again the paths filter makes troubles...
Do you mind re-pushing the commit, but with `skip-checks: true` in the commit message?
Per https://docs.github.com/en/github/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/about-status-checks#skipping-and-requesting-checks-for-individual-commits.

**Comment by adrian17** [comment]
Rebasing on top of master should fix it too, right?

**Comment by midgleyc** [comment]
With #5332 in, rebasing this on `master` should have it run all the checks it needs to.

**Pull request merged**